### ISSUE TITLE
[Design]. Interfacing with S2-LP.

### ISSUE THREAD

**Issue by sinapseenergia** [Sinapse-Energia/IoT-Hub-Home#4]
![image](https://user-images.githubusercontent.com/27274287/51169339-f1665300-18ab-11e9-9f8b-efa9ce5467b3.png)

As shown in above image, when S2-LP is used, only GPIO0 is used because we have not more signals for interfacing with main microcontroller **STM32F215RET**. 

 Question is. **is it possible to develop the firmware for S2-LP with using only the GPIO0 signal?**

**Comment by ralcaide** [comment]
@fpgamcu , could you please answer the question. Check also that HW-FW document uploaded to the project

**Comment by fpgamcu** [comment]
Yes, it is possible, in this task https://github.com/Sinapse-Energia/AP-Sinapse/issues/187 we are using only next pins:
SDO, SDI, SCLK, CSN, SDN, GPIO0
It is minimum required to driver the S2-LP
